inception-recommendation-api/src/main/resources/desc/type/inception-recommendation-api-typesystem.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<typeSystemDescription xmlns="http://uima.apache.org/resourceSpecifier">
+    <name>inception-recommendation-api-typesystem</name>
+    <description/>
+    <version>1.0</version>
+    <vendor/>
+  <types>
+    <typeDescription>
+      <name>de.tudarmstadt.ukp.inception.recommendation.api.type.PredictedSpan</name>
```

Comment:
@Rentier didn't you say we add the original type to the CAS instead of the `PredictedSpan`? Or does that only apply when talking to the DKPro TC external recommender?

Reply:
ping?

Reply:
I do not understand your question.

Reply:
I made it flexible so that a recommender can decide what type he wants to predict. Right now, we let external recommender predict the target annotation, internal recommender predict `PredictedSpan`.

Reply:
I guess that means that right now we don't get confidence scores from the external recommender meaning that it cannot be used with active learning, right?

Reply:
Yes, that is true. But you can specify the feature that the recommender predicts. If we make it configurable via the UI then you could implement an external recommender that uses `PredictedSpan`.